app/assets/javascripts/application/suggest_user.js

```diff
@@ -0,0 +1,55 @@
+const counterFactory = () => {
+    let cnt = 0;
+    return {
+        count() {
+            return ++cnt;
+        },
+        reset(n) {
+            cnt = 0;
+        },
+        get value() {
+            return cnt;
+        }
+    }
+}
+const counter = counterFactory();
+const suggestions = () => document.getElementById('user_suggestions');
+
+function suggestUser() {
```

Comment:
I see in the form that this function is called every time the corresponding input changes. I'd suggest using rxjs's `debounce` to prevent too many requests.

Reply:
Okay, I will look into it.